actors/miner/src/lib.rs

```diff
@@ -4612,11 +4610,15 @@ where
                 continue;
             }
 
+            let deal_weight = BigInt::from(deal_sizes.deal_space);
+            let verified_deal_weight = BigInt::from(deal_sizes.verified_deal_space);
```

Comment:
You have forgotten to multiply size by duration for weight here.

Reply:
simple qa power calculation assumes deal weight means space.  We can't really distinguish on the type level using this approach if sector info has a single type.